help/en/releasenotes/current-draft-note.shtml

```diff
@@ -498,7 +498,9 @@
 
 	    <h4>USS CTC Logic</h4>
             <ul>
-                <li></li>
+                <li>Improve the
+                <a href="http://jmri.org/jython/ctc/TwoColumnMachine.py ">jython/ctc/TwoColumnMachine.py </a>
```

Comment:
There is a space in the URL. Could that have any effect on any browser? Meaning the browser would interpretate the link as:
http://jmri.org/jython/ctc/TwoColumnMachine.py%20

Reply:
Thanks. Fixed.